### PR TITLE
Update activity standards and conditions

### LINKED ISSUE #1523: Implement the Standards and Conditions subsection using the CMS design system
From the design ticket #1465 we are ready to implement!

 One outstanding question I just noticed for @jeromeleecms, @NAretakis , @hbillings, and @mgwalker  - in the mockup they have a WYSIWYG text format editor. I am not sure if this is needed or it can just be a format-less text area.

Final mockups:
image

image

### This task is done when...
 - [ ] All the subsections under Standards and Conditions follow the pattern of theses mockups
 - [ ] Use the [helper text pattern](https://hitech-apd.app.cloud.gov/apd#apd-state-profile-key-personnel) like the one in the Key Personnel section
 - [ ] Designers approve! 
 
Details:
- This design was decided upon on 5/7 as part of our post-standup parking lot session.
- The styles are header, [hint text (like the ones in the form)](https://design.cms.gov/components/text-field/), and then helper text (the one with the light blue background, then text area.
- Don't worry about the exact pixels heights from this mockup, I am using the helper text pattern and that should apply to all of the sections under Standards & Conditions.
  - I also didn't mock up every section because they are looooooong and will all look the same :)
- We may come back to something like a help drawer in v2 and beyond.

### PR DESCRIPTION
**Pull request by hbillings**
Closes #1523.

Note that the design mocks show the "detail" text in gray. I didn't apply that here because it would have meant changing the way all of the instruction text renders.

### This pull request changes...
- Changes the YAML file as far as what is helptext and what is a short instruction for this section.
- Modifies the Instruction component so that you can pass an optional heading level.

### This pull request is ready to merge when...
- [x] Tests have been updated (and all tests are passing)
- [x] This code has been reviewed by someone other than the original author

### This feature is done when...
- [x] Design has approved the experience
- [x] Product has approved the experience

### CODE REVIEW

**Comment by hbillings** [comment]
@jameshupp I made the switch from `short` to `detail` because `short` is bold by default. 😬 I'm sort of not in love with the idea of changing it to change the display, so if you aren't either, I can explore a more targeted CSS solution.

@jeromeleecms  I _think_ I made the required changes to the code to support the rich text editor (thank you for catching that!) but I'd love @mgwalker's eyes on it because holy crow my brain does not get the currying that's happening there.

**Comment by jameshupp** [comment]
Right I forgot about that! Mostly I'm now thinking `short` isn't a very good name for that. No reason to mess with it for the moment, though.

File: web/src/containers/activity/StandardsAndConditions.js [hunk 1]
```diff
@@ -3,19 +3,17 @@ import React, { Component } from 'react';
...
 import { Subsection } from '../../components/Section';
-import { t } from '../../i18n';
 import { selectActivityByKey } from '../../reducers/activities.selectors';
 import { STANDARDS } from '../../util';
 
 class StandardsAndConditions extends Component {
-  handleChange = field => e => {
-    const { value } = e.target;
+  sync = name => html => {
+    console.log('yo');
```
Comment (mgwalker) [inline web/src/containers/activity/StandardsAndConditions.js]: sup

Comment (hbillings) [reply web/src/containers/activity/StandardsAndConditions.js]: AHAHAHAHA ahem sorry thanks

**Comment by hbillings** [comment]
@mgwalker Looks like Snyk found a vuln in Axios. I'm not entirely sure where we're using Axios, or if it's a vulnerability we're concerned about. Halp assess, plz?

**Comment by mgwalker** [comment]
@hbillings Axios is the wrapper/polyfill we use for the fetch API, so it's doing all the requests to the backend for us. We care, but not about this particular vulnerability. The warning should be suppressed in a commit on master, so merging that in should clear it up, I think.  For some reason Github isn't giving me to button to sync with master, though. :yay:

**Pull request merged**